ANC Reference App update (#239)
medic/cht-docs: content/en/apps/examples/anc.md

        task="A [Health Facility ANC Reminder Task](https://docs.google.com/document/d/17pJXBf2gEB2wD1P5g9S5XQ2u4QnLGMiUU4sX0XQsvSk/edit#heading=h.v3b7bata6j) 1 week ahead of the facility visit to remind the woman to attend."
        resolution="CHW submits *Health Facility ANC Reminder Form*, confirming that she called or visited the woman to remind her of her upcoming facility visit."
  %}}
  {{% workflow
        condition="If the CHW notices danger signs at any time, then she submits a *Danger Sign Form* and immediately refers the patient to the facility."
        task="A [Danger Sign Follow-Up Task](https://docs.google.com/document/d/17pJXBf2gEB2wD1P5g9S5XQ2u4QnLGMiUU4sX0XQsvSk/edit#heading=h.82ea7ww1x1k) will appear immediately and is due 3 days later. Tasks persists for 7 days after due date."
        resolution="CHW submits a *Danger Sign Follow-Up Form*, verifying that she called or visited the woman to confirm that she attended the facility. If this is not received, another *Danger Sign Follow-Up Task* is triggered."
  %}}
{{% /workflow-table %}}
 
### Delivery Workflow

{{% workflow-table

>   image-1="delivery-1.png"
>   image-2="delivery-2.png"
>   image-3="delivery-3.png"

Reviewer: ```suggestion
  condition-image="delivery-1.png"
  task-image="delivery-2.png"
  resolution-image="delivery-3.png"
```

---
title: "Antenatal Care"

> linkTitle: "Antenatal Care"

Reviewer: Given that the config includes PNC, I think we should actually be referring to this as a _Maternal and Newborn Health_ app instead. What do you think?

Author: sounds good, updated throughout the text

> linkTitle: "Reference Application: ANC"

Reviewer: A reference app isn't a term that is consistently used in our field, so I am hesitant to make it so prominent in the title here. The term is used with some tools, whereas others use other terms, like "Templates". From an external perspective, what is the difference between an example and reference app? Do we need to make that distinction in the menu structure, or just clarify early in the text what a reference app is? Eg that this example is ready to use as-is and has full set of tests.

Author: Yeah, I was unsure about that as well. Referring to it similarly to the other examples- "Antenatal Care" - with an explanation that it's a reference app, makes sense.

weight: 1
description: >

>   Antenatal care workflows for CHWs using a mobile app

Reviewer: Having the term reference app in the description is useful.

Author: updated

relatedContent: >
  apps/concepts
  apps/reference/forms/app
  apps/reference/tasks
---

{{% pageinfo %}}

> This Antenatal Care workflow is a "reference application" that provides a template for structuring and organizing your Community Health Toolkit digital health app, its configuration code and testing framework. It is a great way to learn about the CHT's foundation for forms, data fields, and analytics that can be easily customized to fit your programs context.

Reviewer: ```suggestion
This "reference application" for maternal and newborn health provides a template for structuring and organizing your Community Health Toolkit digital health app, its configuration and test code. It can be used as is, or serve as a great way to learn about the CHT's foundation for forms, data fields, and analytics that can be easily customized to fit your context.
```

> The Standard Reference Application for antenatal care (ANC) provides a template for structuring and organizing your Community Health Toolkit digital health app, its configuration code and testing framework. The reference app can be deployed as-is or configured to meet unique community care requirements. Built off of ANC care guide best practices, it includes a foundation for CHT forms, workflows, and impact analytics.

Author: updated

{{< see-also page="apps/concepts" title="Concepts that will help you understand how CHT applications are built" >}}

## Problem Being Addressed

> Maternal health is the cornerstone of many community health programs. Pregnancy can be a [vulnerable time](https://www.who.int/health-topics/maternal-health) for women living in communities far from medical care. Additionally, the short window immediately after pregnancy is a critical time for catching life-threatening danger signs for the new mother and baby.

Author: updated

## Solution Overview

> The ANC workflow in this reference app helps ensure that women are safe and supported in a timely manner throughout their pregnancy. Early and timely antenatal care visits and increased coordination between CHWs and clinics lead to an increase in facility-based deliveries, which in turn has a strong correlation to maternal and newborn survival. CHT deployments of ANC workflows are used by health workers and clinical staff to:

Author: updated

- Provide on-time antenatal care visits
- Educate at each stage of pregnancy
- Screen and report danger signs
- Refer and encourage facility delivery
- Improve delivery rates in facilities with skilled care

## Forms Hierarchy

> Once a program [hierarchy]({{< relref "apps/concepts/hierarchy" >}}) of people and places is established, forms are added at different levels. This diagram indicates the forms that can be filled about a person in the app (in this case, family members at the household level), as well as the person/user who will access these forms and make the reports (CHWs at the CHW Area level). Some forms are accessible as actions from the family member’s profile as actions, others from the CHW’s task list as tasks, and some as either. 

Author: updated

![forms-hierarchy.png](forms-hierarchy.png)

{{< see-also page="apps/reference/forms/app" title="Controlling form properties" >}}

## Pregnancy and Delivery Workflows

Next, pregnancy and delivery care workflows are defined to connect form actions and data with people. Detailed documentation to ANC Reference App forms and task schedules are linked from the workflow diagrams below. Accompanying this documentation are tips and insights into the design decisions made along the way, and suggestions for how and where to customize the forms.

[![Pregnancy Workflow](preview-pregnancy-workflow.png)](pregnancy-workflow.pdf)

<br /> 

> [![Delivery Workflow](preview-delivery-workflow.png)](delivery-workflow.pdf)

Reviewer: Given that we are relying on the linked PDF, it is time to convert these to markdown/HTML.

@MaxDiz, can you attempt to do it first with markdown tables? If that doesn't have the desired effect we can then more easily adapt the markdown to a HTML layout, which I can help with.

Author: I messed around with converting to a markdown table yesterday. It unfortunately obscures that it's a workflow (ie it's not clear whether to read down or across). Also not sure how to include the icons which add important context.

What do you think about doing an interim merge with the pdf's, and then opening a new issue to create an html solution?

Reviewer: This isn't a big lift, and I'd prefer we take it on now rather than introducing PDFs that we rely on for links.

Reviewer: @MaxDiz do you have the text markdown version of this? Once you commit that I can then HTML-ify it.

## Getting Started with the ANC Reference Application

The ANC Reference Application can be used ‘out-of-the-box’ to provide care. It is also a great way to learn about the CHT’s structure and workflows to help you understand how to use or modify it for your program needs. Here are a few additional resources to help get you started: 

> - View the [complete reference app configuration](https://github.com/medic/cht-core/tree/master/config/default/forms/app)

Reviewer: The configuration is the top level directory. The `forms/app` directory only has the app forms.
```suggestion
- View the [complete reference app configuration](https://github.com/medic/cht-core/tree/master/config/default/)
```
